stack.yaml

```diff
@@ -7,7 +7,8 @@ resolver: lts-5.6
 packages:
 - '.'
 # Packages to be pulled from upstream that are not in the resolver (e.g., acme-missiles-0.3)
-extra-deps: []
+extra-deps:
+- hspec-expectations-lifted-0.5.0
```

Comment:
It may be worth upgrading to a newer LTS to avoid having to wire in an extra dependency like this.

Reply:
Done.